jobserver/models.py

```diff
@@ -145,5 +183,123 @@ class JobOutput(models.Model):
     )
 
 
+class JobRequest(models.Model):
+    """
+    A request to run a Job
+
+    This represents the request, from a Human, to run a given Job in a
+    Workspace.  The job-runner will create any required Jobs for the requested
+    one to run.  All Jobs, either added by Human or Computer, are then grouped
+    by this object.
+    """
+
+    created_by = models.ForeignKey(
+        "User",
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE,
+    )
+    workspace = models.ForeignKey(
+        "Workspace", on_delete=models.CASCADE, related_name="job_requests"
+    )
+
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Not sure where this comment should be filed but we've agreed *all* `Jobs` should be the responsibility of the `job-runner`, so we should model action_id and backend within `JobRequest`.

Reply:
Done.  The UI form view no longer creates a `Job` instance.  There's some UI tidy up to do around no jobs existing in the drop down of the list page but I'll clean that up in a subsequent PR.